cmd/grype/cli/options/datasources.go

```diff
@@ -19,15 +21,17 @@ var _ interface {
 } = (*externalSources)(nil)
 
 type maven struct {
-	SearchUpstreamBySha1 bool   `yaml:"search-upstream" json:"searchUpstreamBySha1" mapstructure:"search-maven-upstream"`
-	BaseURL              string `yaml:"base-url" json:"baseUrl" mapstructure:"base-url"`
+	SearchUpstreamBySha1 bool          `yaml:"search-upstream" json:"searchUpstreamBySha1" mapstructure:"search-maven-upstream"`
+	BaseURL              string        `yaml:"base-url" json:"baseUrl" mapstructure:"base-url"`
+	RateLimit            time.Duration `yaml:"rate-limit" json:"rateLimit" mapstructure:"rate-limit"`
```

Comment:
TIL `time.Duration` is supported by Viper, calling [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)